pages/code-review.md

```diff
@@ -1,6 +1,143 @@
 ---
 permalink: /code-review/
-title: Code Review
+title: Code Review Guide
 ---
 
-# Code Review
+> A friendly guide for reviewing code--and not each other--at 18f.
+
+(Forked from the excellent Consumer Financial Protection Bureau
+[guide](https://github.com/cfpb/front-end/blob/master/code-reviews.md)). This
+is a **WORK IN PROGRESS**. Help us make it better in the
+[#wg-code-reviews](https://18f.slack.com/messages/wg-code-reviews/) channel!
```

Comment:
since this is a public repo and the Slack channel is not, should we make a comment about that here? Perhaps ask people outside of 18F to submit a GH issue?